autoscheduler/frontend/src/components/Navbar/NavBar.tsx

```diff
@@ -53,15 +45,9 @@ const NavBar: React.SFC = () => {
               </Typography>
             </Button>
           </div>
-
-          <Button
-            color="inherit"
-            onClick={(): void => {
-              window.open('/login/google-oauth2/', '_self');
-            }}
-          >
-            {usersName}
-          </Button>
+          <Typography variant="subtitle1">
```

Comment:
Why wrap it in a `<Typography>`?

Reply:
I believe I was doing that to control the text size. I could move the typography into the button component if it looks weird there.

Reply:
Ok, just move it into the `LogoutButton` component